samples/databus/custom-serializer/Core_6/Shared/MessageWithLargePayload.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using NServiceBus;
+
+//the data bus is allowed to clean up transmitted properties older than the TTBR
+[TimeToBeReceived("00:01:00")]
```

Comment:
The sample text says
 > The FileShareDataBus **does not** remove physical attachments once the message has been processed.

Which seems to contradict this comment